## Ejercicio1 - Utilizar operadores 
aritméticos
### Operadores aritméticos en Python

Python proporciona operadores 
aritméticos comunes para que puedas 
realizar operaciones matemáticas en 
tu código. Estos incluyen las cuatro 
operaciones principales de suma, 
resta, multiplicación y división.

Exploremos cómo podemos crear un 
programa que pueda calcular la 
distancia entre dos planetas. 
Comenzaremos usando dos distancias de 
planetas: Tierra (149.597.870 km) y 
Júpiter (778.547.200 km).

In [1]:
# Crear variables para almacenar las dos distancias
# ¡Asegúrate de quitar las comas!

first_planet = 149597870
second_planet = 778547200

## Realizar la operación

Con los valores obtenidos, es el momento de añadir el código para realizar la operación. Restarás el primer planeta del segundo para determinar la distancia en kilómetros. A continuación, puedes convertir la distancia del kilómetro en millas multiplicándola por 0.621.


In [2]:
# Calcular la distancia entre planetas

distance_km = second_planet - first_planet
print(distance_km)

distance_mi = distance_km * 0.621
print(distance_mi)

628949330
390577533.93


## Prueba tu proyecto

Con el código creado, ejecuta el notebook para obtener el resultado. Deberías recibir lo siguiente:

628949330

390577534

## Ejercicio 2: convierte cadenas en números y usa valores absolutos
#### Crear una aplicación para trabajar con números y entrada de usuario

Con frecuencia, deberás convertir los valores de cadena en números para realizar correctamente diferentes operaciones o determinar el valor absoluto de un número.

Para crear nuestra aplicación, queremos leer la distancia del sol para dos planetas, y luego mostrar la distancia entre los planetas. Haremos esto usando input para leer los valores, int para convertir a entero y luego abs para convertir el resultado en su valor absoluto.

#### Lee los valores

Usando input, agrega el código para leer la distancia del sol para cada planeta, considerando 2 planetas.


In [7]:
# Almacenar las entradas del usuario
first_planet = input('Introduzca la distancia del sol para el primer planeta en KM')
second_planet = input('Introduzca la distancia desde el sol para el segundo planeta en KM')

## Convertir a número

Debido a que input devuelve valores de cadena, necesitamos convertirlos en números. Para nuestro ejemplo, usaremos int


In [8]:
# Convierte las cadenas de ambos planetas a números enteros
first_planet = int(first_planet)
second_planet = int(second_planet)

## Realizar el cálculo y convertir a valor absoluto

Con los valores almacenados como números, ahora puedes agregar el código para realizar el cálculo, restando el primer planeta del segundo. Debido a que el segundo planeta podría ser un número mayor, usarás abs para convertirlo a un valor absoluto. También agregarás el código para mostrar el resultado en millas multiplicando la distancia del kilómetro por 0.621


In [9]:
# Realizar el cálculo y determinar el valor absoluto
distance_km = second_planet - first_planet
print(distance_km)

# Convertir de KM a Millas
distance_mi = distance_km * 0.621
print(abs(distance_mi))

628949330
390577533.93
